e2e/studio/features/auth-users.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { expect, Page } from '@playwright/test'
+import { test } from '../utils/test.js'
+import { toUrl } from '../utils/to-url.js'
+import { waitForApiResponse } from '../utils/wait-for-response.js'
+import { createUserViaUI, deleteUserViaUI } from '../utils/auth-helpers.js'
+
+test.describe('auth users list refresh', () => {
+  test.beforeEach(async ({ page, ref }) => {
+    await page.goto(toUrl(`/project/${ref}/auth/users`))
+
+    // Wait for the page to load by checking for the "Users" heading
+    await expect(page.getByRole('heading', { name: 'Users', level: 3 })).toBeVisible()
+
+    // Wait for initial users list to load
+    await waitForApiResponse(page, 'platform/pg-meta', ref, 'query?key=')
+  })
+
+  test('should automatically refresh users list after creating a user', async ({ page, ref }) => {
+    const testEmail = `test-create-${Date.now()}@example.com`
+    const testPassword = 'testpassword123'
+
+    // Create user via UI
+    await createUserViaUI(page, ref, testEmail, testPassword)
+
+    // Verify the user appears in the table WITHOUT manually refreshing the page
+    const userRow = page.getByRole('row').filter({ hasText: testEmail })
+    await expect(
+      userRow,
+      'User should appear in the table immediately after creation without manual refresh'
+    ).toBeVisible({ timeout: 10_000 })
+
+    // Verify the user details are correct
+    await expect(userRow.getByText(testEmail)).toBeVisible()
+    await expect(userRow.getByText('Email')).toBeVisible()
+
+    // Clean up: delete the user
+    await deleteUserViaUI(page, ref, testEmail)
+
+    // Verify the user is removed from the table
+    await expect
+      .poll(async () => {
+        return await page.getByRole('row').filter({ hasText: testEmail }).count()
+      }, 'User should be removed from the table after deletion')
+      .toBe(0)
+  })
+
+  test('should automatically refresh users list after deleting a user', async ({ page, ref }) => {
```

Comment:
the setup/teardown requirements make this test exactly the same as the previous test, unless I'm missing something. can we just consolidate them?

Reply:
🙈 omg its exactly the same test. Removed